publishing-rules.yaml

```diff
@@ -3,6 +3,10 @@ skip-gomod: true
 rules:
 - destination: kubernetes
   branches:
+  - name: origin-4.3-kubernetes-1.16.0-rc.2
```

Comment:
is this the name of the branch? - https://github.com/openshift/kubernetes/tree/origin-4.3-kubernetes-1.16.0-rc.2
where do I find definitions for the rules?